examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/weight_only/run_clm_no_trainer.py

```diff
@@ -77,58 +91,43 @@
 parser.add_argument('--gptq_max_seq_length', type=int, default=2048,
                     help='Calibration dataset sequence max length, '
                         'this should align with your model config, '
-                        'and your dataset builder args: args.pad_max_length')
+                        'and your dataset builder args: args.pad_max_length.')
+
 # =============AWQ configs====================
 parser.add_argument("--use_auto_scale", action="store_true",
                     help="Enables best scales search based on activation distribution.")
 parser.add_argument("--use_auto_clip", action="store_true",
-                    help="Enables clip range searchc.")
+                    help="Enables clip range search.")
 parser.add_argument("--folding", action="store_true",
                     help="Allow insert mul before linear when the scale cannot be absorbed by last layer for TEQ/AWQ.")
 parser.add_argument('--absorb_layer_dict', type=dict, default={},
                     help="The layer dict that scale can be absorbed for TEQ/AWQ.")
+
 # ============AUTOROUND configs==============
-parser.add_argument(
-    "--lr",
-    type=float,
-    default=None,
-    help="learning rate, if None, it will be set to 1.0/iters automatically",
-)
-parser.add_argument(
-    "--minmax_lr",
-    type=float,
-    default=None,
-    help="minmax learning rate, if None,it will beset to be the same with lr",
-)
-parser.add_argument("--autoround_iters", default=200, type=int, help="num iters for autoround calibration.")
-parser.add_argument("--autoround_nsamples", default=128, type=int, help="num samples for autoround calibration.")
-parser.add_argument(
-    "--disable_quanted_input",
-    action="store_true",
-    help="whether to use the output of quantized block to tune the next block",
-)
+parser.add_argument("--lr", type=float, default=None,
+                    help="Learning rate, if None, it will be set to 1.0/iters automatically")
+parser.add_argument("--minmax_lr", type=float, default=None,
+                    help="Minmax learning rate, if None,it will beset to be the same with lr")
+parser.add_argument("--Autoround_iters", default=200, type=int,
```

Comment:
this should be lowercase --autoround_iters

Reply:
fixed https://github.com/intel/neural-compressor/pull/2053/commits/c818087d01c38a97fe2a81bf8c2ac2dbb04b2cfa